pulp_ansible/app/viewsets.py

```diff
@@ -229,6 +233,31 @@ class CollectionDeprecatedViewSet(ContentViewSet):
     serializer_class = CollectionSerializer
 
 
+class AnsibleNamespaceFilter(ContentFilter):
+    """
+    A filter for namespaces.
+    """
+
+    class Meta:
+        model = AnsibleNamespace
+        fields = {
+            "name": NAME_FILTER_OPTIONS,
+            "company": NAME_FILTER_OPTIONS,
+            "metadata_sha256": ["exact", "in"],
+        }
+
+
+class AnsibleNamespaceViewSet(ReadOnlyContentViewSet):
```

Comment:
So this is the viewset for the Pulp Namespace endpoint found at `/pulp/api/v3/content/ansible/namespace/`. ReadOnly makes it so the only two endpoints exposed are `list` and `retrieve`. The Galaxy V3 Plugin Namespace endpoint has the full list of actions `list`, `retrieve`, `create`, `update`, `delete`. 

The main reason is because this model is closely tied to being apart of a repository, so the Pulp endpoints will mainly be a global view endpoint to see what is in the system while the Galaxy endpoint will be what is actually used to modify Namespaces in a repository. At least that is how I envisioned it working, if there are other requirements we can change it.